packages/js-sdk/src/ConstantFlowAgreementV1Helper.d.ts

```diff
@@ -1,6 +1,7 @@
 import { Transaction } from "web3-core";
-import Framework from "./Framework";
+import type { Framework } from "./Framework";
 import type { LoadedContract } from "./loadContracts";
+import type BN from 'bn.js';
```

Comment:
can we use `import type { BigNumber } from "ethers";`? 
there is a slight difference from `bn.js`: https://docs.ethers.io/v5/api/utils/bignumber/#BigNumber--BigNumber--notes--why-not-bignumber-js-bn-js-bigdecimal-etc

however, this may break something with web3, but I think if we can use `BigNumber` it is more suitable.